cmd/neofs-node/tree.go

```diff
@@ -3,10 +3,15 @@ package main
 import (
 	"context"
 
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
 	"github.com/nspcc-dev/neofs-node/pkg/services/tree"
 )
 
 func initTreeService(c *cfg) {
+	if !config.BoolSafe(c.appCfg.Sub("tree"), "enabled") {
```

Comment:
just wanna mention: is it ok that admins gonna see "initializing tree service..." even if it's not enabled?

that question could also be applied to some other services, did not check

Reply:
No, added a log message.